RFD.md

```diff
@@ -0,0 +1,191 @@
+---
```

Comment:
Can you also share your `.proto` file?

Reply:
added `.proto` definition to the RFD, can include full separate file if preferred